libraries/WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -146,21 +146,20 @@ class StaticRequestHandler : public RequestHandler {
     return true;
   }
 
-  bool handle(WebServer &server, HTTPMethod requestMethod, String requestUri) override {
+  bool handle(WebServer &server, HTTPMethod requestMethod, const String& requestUri) override {
     if (!canHandle(server, requestMethod, requestUri)) {
       return false;
     }
 
     log_v("StaticRequestHandler::handle: request=%s _uri=%s\r\n", requestUri.c_str(), _uri.c_str());
 
     String path(_path);
-    String eTagCode;
```

Comment:
moving it changes nothing

Reply:
Less allocated on the stack when it returns early (e.g. due to the added single recursive call you asked about in the previous comment.